compiler/src/utils/version.re

```diff
@@ -0,0 +1,72 @@
+// A really bad version parsing library
```

Comment:
😤

Reply:
But honestly after reading through it, I really don't think it's all that bad! The only feature that I see is missing is support for alphas/betas/rcs. Otherwise looks fairly feature complete to me.

Reply:
I changed the comment to "moderately bad" 😛 I actually wrote the comment when it was a "Really bad semver library" but then realized that I didn't need semver and just needed version compares.